modin/core/execution/ray/common/utils.py

```diff
@@ -219,11 +204,25 @@ def initialize_ray(
             if not GPU_MANAGERS:
                 for i in range(GpuCount.get()):
                     GPU_MANAGERS.append(GPUManager.remote(i))
+    else:  # ray is already initialized, check runtime env config
+        main_cfg = ray.worker.global_worker.core_worker.get_job_config()
+        try:
+            runtime_env = json.loads(main_cfg.runtime_env_info.serialized_runtime_env)
+        except json.JSONDecodeError:
+            runtime_env = {}
+        env_vars = runtime_env.get("envVars", {})
+        for varname, varvalue in extra_init_kw["runtime_env"]["env_vars"].items():
+            if str(env_vars.get(varname, "")) != str(varvalue):
+                ErrorMessage.single_warning(
+                    "If initialising Ray yourself, please ensure its runtime env "
+                    + f"sets environment variable {varname} to {varvalue}"
+                )
+
     _move_stdlib_ahead_of_site_packages()
     ray.worker.global_worker.run_function_on_all_workers(
         _move_stdlib_ahead_of_site_packages
     )
```

Comment:
It seems to me we had kind of a similar problem with these lines. I wonder if we will have to add an additional config for ray init, when Ray removes `run_function_on_all_workers`?

Reply:
This may not be a problem anymore (Python 3.6+). Previously when users had `typing` pypi package installed it would create conflicts with the stdlib `typing`.

Reply:
Should I drop those lines then?

Reply:
It's ok with me to drop.

Reply:
Thank you!